### PR TITLE
Add `plugins.json` file + schema

### LINKED ISSUE #66: Define structured way to host plug-in meta-data
__What shall we do__

As a integration author I love to discover available plug-ins in a structured manner. At the same time I'd like to understand an individual plug-ins context (i.e. compatibility) in a structured manner (currently represented as a [compat table](https://github.com/bpmn-io/bpmn-js-token-simulation-plugin#compatibility-notice) + badge)), too.

__Why shall we do it__

Providing both in a structured way allows us to build tooling for plug-in discovery, installation, and upgrading (i.e. [`mopl`](https://github.com/pinussilvestrus/mopl), just without crawling).

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #66 

This provides a simple start for an accessible, structured plugins file that consumers can use to build their tools on top. Cf. [to this comment](https://github.com/camunda/camunda-modeler-plugins/issues/66#issuecomment-1128805330) for some rationale.

It also adds a simple GitHub action to verify the JSON against a schema (we use `ajv-cli` for it).

Failed build example: https://github.com/camunda/camunda-modeler-plugins/runs/6502985121?check_suite_focus=true
Success build example: https://github.com/camunda/camunda-modeler-plugins/actions/runs/2350471266

Note that I set `additionalProperties` to false for the plugins items schema, to have more control about what we deliver. I'm open to discussing this.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+    - name: Install ajv-cli
+      run: npm -g i ajv-cli
+    - name: Validate Plugins JSON File
+      run: ajv test -s resources/schema.json -d plugins.json --errors=text --valid
```
Comment (nikku) [inline .github/workflows/CI.yml:32]: Consider renaming schema to `plugins-schema.json` or something.

**Review by nikku — COMMENTED** [review]
Great stuff all together. Just one tiny remark.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (pinussilvestrus) [reply .github/workflows/CI.yml]: Done 👍

**Comment by nikku** [comment]
:clap:

**Pull request merged**